src/decisionengine/framework/engine/DecisionEngine.py

```diff
@@ -613,6 +613,33 @@ def rpc_set_channel_log_level(self, channel, log_level):
             worker.task_manager.set_loglevel_value(log_level)
         return f"Log level changed to : {log_level}"
 
+    def rpc_get_source_log_level(self, source):
+        with self.source_workers.access() as workers:
+            worker = workers.get(source)
+            if worker is None:
+                return f"No source found with the name {source}."
```

Comment:
In the new `rpc_get/set_source_log_level` methods, there are some warnings about code not being covered by tests.  Is it possible to add some unit tests for these?

Reply:
Or is that the "NOTSET" bit in the client-server unit test?

Reply:
The warnings show up if you go the PR and click on the "Files Changed" tab.  You'll see this conversation, for example, and right above it an example of a Codecov warning that I'm referring to.

Reply:
I added more unit tests to address most of the warnings.
- I cannot cover line 137 in `src/decisionengine/framework/engine/SourceWorkers.py` due to the fact that I cannot create a queue handler in testing mode.